### ISSUE TITLE
AttributeError on notmuch database object

### ISSUE THREAD

**Issue by maxking** [gauteh/gmailieer#28]
```
Traceback (most recent call last):
  File "/usr/local/bin/gmi", line 8, in <module>
    g.main ()
  File "/home/maxking/Documents/gmailieer/lieer/gmailieer.py", line 136, in main
    args.func (args)
  File "/home/maxking/Documents/gmailieer/lieer/gmailieer.py", line 307, in pull
    self.full_pull ()
  File "/home/maxking/Documents/gmailieer/lieer/gmailieer.py", line 538, in full_pull
    (rev, uuid) = db.get_revision ()
AttributeError: 'Database' object has no attribute 'get_revision'
```

Not sure what this means but I was trying to create a fresh maildir using gmailieer. I have python3.5.2 on Linux Mint 18.1.

**Comment by maxking** [comment]
It looks like `Database` object has a `get_version` method which returns only an `int` object which is rev.

I can send in a pull request to fix this if you are Ok with it?

**Comment by julian-klode** [comment]
You need current git of notmuch for it to work, that adds get_revision.

**Comment by maxking** [comment]
Works like a charm! Thanks very much!

**Comment by maxking** [comment]
Maybe add this information to documentation till the next version of python-notmuch comes out?

**Comment by gauteh** [comment]
Abhilash Raj writes on juni 7, 2017 4:42:
> Maybe add this information to documentation till the next version of python-notmuch comes out?
> 

Sounds good, please send a PR if you have the time.

**Comment by maxking** [comment]
I will just keep it open so that I remember to send a PR for the documentation, hope that is OK!

**Comment by gauteh** [comment]
Abhilash Raj writes on juni 8, 2017 3:20:
> I will just keep it open so that I remember to send a PR for the documentation, hope that is OK!

Sure!

**Comment by gauteh** [comment]
Fixed in #31.
